src/style.css

```diff
@@ -1 +1,87 @@
 /* add styles here */
+@font-face {
+  font-family:Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family:Avenir;
+  font-style: normal;
+  font-weight: 400;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family:Avenir,sans-serif;
+  font-weight: 300;
+}
+
+.form-big {
+  margin-top: 20px;
+}
+
+.input-form-big {
+  background-image:url(/src/images/Search.svg);
+  background-repeat:no-repeat;
+  background-size: 19px 19px;
+  background-position: top 25px left 20px ;
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
```

Comment:
pls use one color format in the entire project. either rgb of hex

Reply:
isn't fixed